fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyAdapter.java

```diff
@@ -174,12 +181,12 @@ public boolean doesTableExist(String schemaName, String tableName) {
 
     @Override
     public void createSequence(String schemaName, String sequenceName, int cache) {
-        /*CREATE SEQUENCE fhir_sequence
-             AS BIGINT
-     START WITH 1
-          CACHE 1000
-       NO CYCLE;
-    */
+        /* CREATE SEQUENCE fhir_sequence
+         *     AS BIGINT
+         *     START WITH 1
+         *     CACHE 1000
+         *     NO CYCLE;
+        */
         // Derby doesn't support CACHE
         final String sname = DataDefinitionUtil.getQualifiedName(schemaName, sequenceName);
         final String ddl = "CREATE SEQUENCE " + sname + " AS BIGINT START WITH 1 NO CYCLE";
```

Comment:
I didn't change this to 1000, should we change to 1000 here?

Reply:
to align with db2

Reply:
no opinion on this one...i forget why you said it was better to start at 1000 even for db2